AEPIdentity/Sources/Identity.swift

```diff
@@ -58,7 +60,7 @@ import Foundation
         guard let state = state else { return false }
 
         // fast boot identity without waiting for configuration
-        state.boot(event: event, createSharedState: createSharedState(data:event:))
+        state.boot(createSharedState: createSharedState(data:event:))
```

Comment:
I don't think this is needed as you already do it in the `onRegistered` method.

Reply:
Yes, I was thinking about the case where state is null onRegistered but yes then it would be null everywhere so doesn't matter. I will remove it.